Milad.css

```diff
@@ -0,0 +1,127 @@
+/* Reset some default styles */
+body, h1, h2, h3, p, ul, li {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Comic sans MS, sans-serif;
+    background-color: #f0f0f0;
+    background-image: url(https://www.w3schools.com/css/lights600x400.jpg);
+  background-repeat: no-repeat;
+  background-size:  cover;
+  color: #FFFFFF;
+  height: 700px;
+  width: auto;
+  
+      
+      
```

Comment:
I recommend removing all unnecessary spaces in this file to enhance readability.

Consider using an extension in VSCode called **"Prettier - Code formatter."** This extension will format your code in a more readable way.